metricbeat/module/docker/network/_meta/data.json

```diff
@@ -41,23 +53,247 @@
                 "packets": 0
             },
             "inbound": {
-                "bytes": 23047,
+                "bytes": 92092613,
                 "dropped": 0,
                 "errors": 0,
-                "packets": 241
+                "packets": 424322
             },
             "interface": "eth0",
```

Comment:
Metrics in these documents are per interface, but the new metrics are per container. We may need to create a new `network_summary` metricset as we had to do in the `system` module.